packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -704,6 +761,7 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
           CpuProfileData.nameKey: verboseName,
           CpuProfileData.categoryKey: category,
           CpuProfileData.resolvedUrlKey: rawUrl,
+          CpuProfileData.resolvedPackageUriKey: packageUriWithSourceLine,
```

Comment:
add the packageUri and the  sourceLine to the json separately.

Reply:
oops ty!

Reply:
I've fixed this up and I'll leave the sourceline fix for the follow up issue:
https://github.com/flutter/devtools/issues/3802